src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const list = [...document.querySelectorAll('li')];
+const ulElement = document.querySelector('ul');
+
+function getSalary(person) {
+  return Number(person.dataset.salary.slice(1).replace(',', ''));
+}
+
+function sortList(people) {
+  const sorted = people.sort((prevPerson, currPerson) => (
```

Comment:
Don't change input params
```suggestion
  const sorted = [...people].sort((prevPerson, currPerson) => (
```